src/experimentConfig.ts

```diff
@@ -74,7 +83,7 @@ const experimentConfigPromise: Promise<ExperimentConfig> = (async (): Promise<Ex
   } catch (err) {
     // Not found or incompatible. Re-generate with random values according to
     // the defined distribution.
-    const generatedExperimentConfig = asExperimentConfig({})
+    const generatedExperimentConfig = asExperimentConfig(generateExperimentConfigVal)({})
```

Comment:
Why all the complexity of creating a function that gets passed in when all you really care about is a boolean like `useDefaults`. Pass that as the first arg to `asExperimentConfig` and you don't need to check for a null function.